examples/crowd-funding/src/contract.rs

```diff
@@ -220,8 +300,9 @@ impl CrowdFunding<ViewStorageContext> {
     async fn cancel_campaign(&mut self) -> Result<(), Error> {
         ensure!(!self.status.get().is_complete(), Error::Completed);
 
+        #[cfg(not(any(test, feature = "test")))]
```

Comment:
(In the long run we should allow setting the time in tests.)

Reply:
yeah this is not great. Let me create a task